src/App.tsx

```diff
@@ -1,12 +1,112 @@
-import './App.css'
+import { ChangeEvent, useEffect, useState } from 'react';
+import { Alert, Button, Checkbox, Input, Text, Radio, colors, Icon, useTheme } from '@alanpinhon/deep-feels-ui';
+import './App.css';
 
 function App() {
 
+  const {theme, setTheme, setUseDeviceTheme} = useTheme();
+  const [className, setClassName] = useState('');
+  const [isChecked, setIsChecked] = useState(false);
+  const [inputValue, setInputValue] = useState('');
+  const [checkRadio, setCheckRadio] = useState(localStorage.getItem('useDeviceTheme') === 'true' ? 'device' : theme);
+
+  //Theme
+  useEffect(() => {
+    const bodyStyle = document.body.style;
+    (theme === 'dark')
+      ? bodyStyle.setProperty('background-color', colors.lightPurple[800])
+      : bodyStyle.setProperty('background-color', colors.lightPurple[50])
+  }, [theme]);
+
+
+   // Alert
+  const setAlertTime = () => {
+    setClassName('active');
+    setTimeout(() => {
+      setClassName('');
+    }, 3000);
+  };
```

Comment:
Esto debería estar dentro del componente Alert. No creo que debamos mostrar la alerta con base a una clase. Tal vez necesitaríamos mostrarlo mediante un react portal y retornar `null` cuando se cierre el `Alert`. Todo eso debe hacerse dentro del componente Alert

Puedes mantener la clase `.active` pero esa iría al momento de montar el componente. Y al desmontarlo, puedes cambiar la clase y poner el setTimeout para despues desmontar por completo el componente para que no estemos ocupando memoria inecesariamente. Algo en plan:


```tsx
const Alert = () => {
const [visible, setVisible] = useState(props.visible || true)
const [class, setClass] = useState("active");

const onClose = () => {
 setClass("hide");
 setTimeout(() => setVisible(false), 3000); // Unmounting the component after the CSS animation has been completed to prevent making state updates of an unmounted component
}

return visible ? React.createPortal(<div role="alert className={class}>{...}</div>) : null
}
```